spacy_llm/backends/integration/hf/base.py

```diff
@@ -16,16 +18,19 @@ class HuggingFaceBackend(abc.ABC):
     def __init__(
         self,
         model: str,
-        config: Dict[Any, Any],
+        config: Dict[str, Dict[Any, Any]],
```

Comment:
I had it like that at first, but thought this would be cleaner. I'm open to changing this though. Why do you prefer `init` and `run` configs?

Reply:
I like the fact that you don't have to resort to some exotic type hints and validation. It seems clearer to me to convey the constraints directly rather than through a nested dictionary. Bottom line it's largely based on aesthetics and subjective taste :)

Reply:
We'd need to explain this in the bwd-compat section of the 0.3.0 release notes, then.